aidants_connect_habilitation/static/js/organisation_form.js

```diff
@@ -6,12 +6,25 @@
     Object.assign(DynamicForm.prototype, {
         "connect": function connect() {
             this.selectOrgType(this.typeInputTarget.value);
+            this.showHide(this.onlyShownIfPrivateOrg, this.privateOrgInput.checked);
```

Comment:
Ici ça bug parce que `onlyShownIfPrivateOrg` est `undefined` et je n'ai pas compris comment on fait pour qu'il ne soit pas `undefined`. @christophehenry halp

Reply:
À l'intérieur du contrôleur, les [targets s'appellent `this.[name]Target`](https://stimulus.hotwired.dev/reference/targets#properties). Donc ici, `this.onlyShownIfPrivateOrgTarget`. Et il faut aussi que [tu aies signalé son existance au contrôleur](https://stimulus.hotwired.dev/reference/targets#definitions) en rajoutant `"onlyShownIfPrivateOrg"` à `DynamicForm.targets`.

Reply:
Et pareil pour `privateOrgInput`.